common/laws/00_church_and_state.txt

```diff
@@ -78,6 +78,17 @@ law_state_religion = {
 			}
 			add = 750
 		}
+		if = {
+			limit = { 
+				has_journal_entry = je_sick_man_absolute_islam
+				OR = {
+					has_law = law_type:law_freedom_of_conscience
+					has_law = law_type:law_total_separation
+					has_law = law_type:law_state_atheism
+				}
```

Comment:
unnecessary qualifier, this is always the case. Or is some kind of super religion law in the works?

Reply:
What do you mean it's unnecessary? If the Ottomans were on any of these laws, their desire to pass state religion is much higher. That's how all of these work.